gen/dcos-config.yaml

```diff
@@ -2440,6 +2447,13 @@ package:
         ## Apply DC/OS component name tag according to the documentation.
         [inputs.prometheus.tags]
           dcos-component-name = "Admin Router Agent"
+      # Read metrics from DC/OS Diagnostics Prometheus endpoint.
+      [[inputs.prometheus]]
+        ## An array of urls to scrape metrics from.
+        urls = ["unix:///var/run/dcos/dcos-diagnostics.sock?path=/metrics"]
+        ## Apply DC/OS component name tag according to the documentation.
+        [inputs.prometheus.tags]
+          dcos-component-name = "DC/OS Diagnostics"
```

Comment:
I noticed that the admin router metrics are tagged differently form master vs agents, not sure if that was something you also want to do here? `Admin Router` vs `Admin Router Agent`

Reply:
It's OK to have the same tag both on master and agents.